cmd/aries-agent-mobile/pkg/api/api.go

```diff
@@ -38,4 +38,7 @@ type AriesController interface {
 
 	// GetKMSController returns an implementation of KMSController
 	GetKMSController() (KMSController, error)
+
+	// RegisterNotifier associates a notifier to a topic
+	RegisterNotifier(n Notifier, topic string, payload []byte) error
```

Comment:
I'll remove `payload`.

For topic, type `[]string` won't get parsed by gomobile. The easiest approach will be for the mobile app to send the topics as a JSON array in a `string`.

An alternative is to create a method called `AppendTopic(topic string)` on the `Notifier` interface which will require implementations to have a string array instance variable. To then access this variable, they would also have to implement a `GetTopics() string` method which still has to return a JSON array as a `string` because of gomobile.

Reply:
how about that topics should be comma-separated

Reply:
ok that's better. i'll add a comment for this